app/experimenter/experiments/constants/nimbus.py

```diff
@@ -432,13 +432,24 @@ class NimbusTargetingConfig:
 TARGETING_CORE_USER_URIS = NimbusTargetingConfig(
     name="Core user (uris)",
     slug="core_user_uris",
-    description="More than 20 days of activity in the past 28 days",
+    description="At least 21 days of activity in the past 28 days",
     targeting="userMonthlyActivity|length >= 21 && "
     "(currentDate|date - profileAgeCreated|date) / 86400000 >= 28",
     desktop_telemetry="",
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+TARGETING_CORE_USER_NEED_PIN = NimbusTargetingConfig(
+    name="Core user (need pin)",
+    slug="core_user_need_pin",
+    description="At least 21 days of activity in the past 28 days needing pin",
+    targeting="({core_user} && doesAppNeedPin || {sticky})".format(
```

Comment:
I see #7120 wraps with parens, so maybe I should drop these (and the other places where we explicitly added them?)

Reply:
Yeah you don't need the outermost parens, those'll get added later, but do you want parents around the `&&` clause so both operands are `||`ed with the sticky clause?

Reply:
I removed the unnecessary parens from the activity level targeting

Reply:
To be clear, the `core && pin || sticky` is the same as `(core && pin) || sticky`, so parens aren't necessary but can be added. Most of the activity level targeting does not have the unnecessary parens.